supabase/config.toml

```diff
@@ -195,6 +195,12 @@ url = ""
 # If enabled, the nonce check will be skipped. Required for local sign in with Google auth.
 skip_nonce_check = false
 
+[auth.external.github]
+enabled = true
+client_id = "Ov23liLoN4ZwUE1whqJv"
```

Comment:
I think this file is fine as is since we aren't using it for production (we just manually edited the Supabase dashboard for that).

In the future maybe we will want to have everything configured through here (using `env(<variable>)` to streamline things.